src/sysext/sysext.c

```diff
@@ -1301,6 +1303,14 @@ static int mount_overlayfs_with_op(
         if (r < 0)
                 return log_error_errno(r, "Failed to make directory '%s': %m", meta_path);
 
+        atfd = open(meta_path, O_DIRECTORY|O_CLOEXEC);
+        if (atfd < 0)
+                return log_error_errno(atfd, "Failed to open directory '%s': %m", meta_path);
```

Comment:
first arg needs to be errno, not atfd, as libc `open()` returns errors via that.